wiki/Tournaments/oPHF/2018/en.md

```diff
@@ -0,0 +1,220 @@
+---
+tags:
+- oPHF 2018
+- oPHF2018
+---
+# osu!Philippines Holiday Friendlies 2018
+
+![oPHF2018 Logo](img/logo.png)
+
+> It's now time for some Holiday cheer!
+
+The **osu!Philippines Holiday Friendlies 2018** is a semi-restricted, semi-open Tournament Series hosted by the osu!PH community for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes. This tournament aims to provide up and coming players a competitive environment where they have a chance at glory themselves. It is the 2nd iteration of the osu!Philippines Holiday Friendlies.

```

Comment:
The **osu!Philippines Holiday Friendlies 2018** (***oPHF2018***) is a semi-restricted, semi-open tournament series for the osu!standard, osu!taiko, osu!catch, and the osu!mania game modes hosted by the osu!PH community. This tournament aims to provide up and coming players a competitive environment where they have a chance to glory themselves. It is the 2nd installment of the osu!Philippines Holiday Friendlies.